src/lib/mongoBackend/dbConstants.h

```diff
@@ -61,6 +61,7 @@
 #define REG_FORMAT                   "format"
 #define REG_DESCRIPTION              "description"
 #define REG_STATUS                   "status"
+#define REG_FORWARDING_INFORMATION   "forwardingInformation"
```

Comment:
This field is not actually used. Se data model analysis here: https://github.com/telefonicaid/fiware-orion/issues/3008#issuecomment-336486377

Reply:
Sure, will be removed.

Reply:
Removed